src/pages/api/auth/syncUserInfo.ts

```
@@ -0,0 +1,71 @@
+    });
+
+    if (!token?.email || !token?.name) {
+      return res.status(400).json({ error: "사용자 정보를 찾을 수 없습니다." });
+    }
+
+    const resultList = await getUserListByEmail(token.email);
+
+    // DynamoDB에 유저가 있음
+    if (resultList.data && resultList.data.length) {
+      const userData = resultList.data[0];
+      // 아직 인증되지 않은 유저라면
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)